examples/wasm/iwasmt.c

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2019 Intel Corporation.  All rights reserved.
+ * Copyright (C) 2020 TU Bergakademie Freiberg Karl Fessel
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+//
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdint.h>
+
+#include <string.h>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wstrict-prototypes"
+#include "wasm_export.h"
+#pragma GCC diagnostic pop
+
+#include <thread.h>
+
+/*provide some test program*/
+#include "blob/main.wasm.h"
+
+
+#define DEFAULT_THREAD_STACKSIZE (6 * 1024)
+#define DEFAULT_THREAD_PRIORITY 50
+
+static int app_argc;
+static char **app_argv;
+
+static void*
+app_instance_main(wasm_module_inst_t module_inst)
+{
+    const char *exception;
+
+    wasm_application_execute_main(module_inst, app_argc, app_argv);
+    if ((exception = wasm_runtime_get_exception(module_inst))){
+        puts(exception);
+    }
+    return NULL;
+}
+
+
+void *
+iwasm_t(void *arg1)
+{
+    wasm_module_t wasm_module = (wasm_module_t) arg1;
+    wasm_module_inst_t wasm_module_inst = NULL;
+    char error_buf[128];
+
+    /* instantiate the module */
+    if (!(wasm_module_inst = wasm_runtime_instantiate(wasm_module, 8 * 1024,
+        8 * 1024, error_buf, sizeof(error_buf)))) {
```

Comment:
```suggestion
    wasm_module_inst = wasm_runtime_instantiate(wasm_module, 8 * 1024, 8 * 1024,
                                                error_buf, sizeof(error_buf));
    if (!wasm_module_inst) {
```

Style is subjective, but still IMO this is easier to read.